frontend/src/components/RFI/RFILanding.vue

```diff
@@ -686,7 +635,7 @@
                   color="#003366"
                   @click="addObjToList(wageObj, model.wageList)"
                 >
-                  Add
+                  Add Wage
```

Comment:
Do we have requirement for this change? If not, I think we should confirm with the business